### DISCUSSION TITLE
open program in another session by gsudo?

### DISCUSSION THREAD

**Discussion by lazna** [gerardog/gsudo#210] [Q&A] [answered]
regarding to [this issue](https://superuser.com/questions/1753451/supress-console-window-appear-on-periodical-schedule-program-runnig-tests), is it possible to start program in different Session? Possibly with cached credentials, to not need enter it every time?

**Comment by gerardog** [comment, marked as answer]
Creating processes in another windows session is out of the scope of gsudo.

**Reply by lazna** [reply]
thanks for answer
